README.md

```diff
@@ -97,6 +97,58 @@ fastify.register(async function (fastify) {
 fastify.listen({ port: 3000 })
 ```
 
+### Custom Fastify hook name
+
+By default, `@fastify/cors` adds a `onRequest` hook where the validation and header injection are executed. This can be customized by passing `hookName` in the options.
+
+```js
+import Fastify from 'fastify'
+import cors from '@fastify/cors'
+
+const fastify = Fastify()
+await fastify.register(cors, { 
+  hookName: 'preHandler',
```

Comment:
We should call it not hookName but simply hook as we do it in other plugins, like in fastify-rate-limit